intro-python/part2/fortune_cookie.py

```diff
@@ -21,7 +21,7 @@ def generate_fortune() -> str:
 def generate_lucky_numbers(how_many: int) -> list:
```

Comment:
I believe that the `how_man: int` in this line is what transforms `how_many` into an `int` so I'm surprised that it's necessary again in line 24. Did something change in the Python syntax perhaps?